backend/src/main/java/com/beaplab/BeaplabEngine/controller/UserController.java

```diff
@@ -130,6 +130,31 @@ public ResponseEntity<UserDto> get(@PathVariable("id") String id) {
         return new ResponseEntity<UserDto>(userDto, HttpStatus.OK);
     }
 
+    /**
+     * handles a GET request for retrieving a user account name, first name, last name
+     * @param id
+     * @return ResponseEntity<String>
+     */
+    @PreAuthorize("hasRole('ROLE_ADMIN') or hasRole('ROLE_USER')")
+    @RequestMapping(value = "/rest/beapengine/user/{id}/accountName", method = RequestMethod.GET, produces = "application/json")
```

Comment:
I think this is a more appropriate route to adhere to REST standards
```suggestion
    @RequestMapping(value = "/user", method = RequestMethod.GET, produces = "application/json")
```